src/pimp_base.h

```diff
@@ -16,6 +17,7 @@ typedef unsigned int       u32;
 #endif
 typedef   signed long long s64;
 typedef unsigned long long u64;
+#endif
```

Comment:
Why do you need to do this?

Reply:
The libgba headers define these types already, and guard it with that macro. All this does is making sure that these don't get redefined if the order of the included headers is 'bad'.

Reply:
`pimp_base.h` is in src, so it's not meant to be included by 3rd party code; it's not part of the API. So I don't think there's any risk of this happening at the moment.

If we want to make it safer to include these, we should simply change to use `stdint.h`-definitions, I think.

Reply:
That'd probably be a better idea in the long run, but I'll roll it back for now.